### DISCUSSION TITLE
After GKE cluster update, kafka is not working.

### DISCUSSION THREAD

**Discussion by nevenara** [strimzi/strimzi-kafka-operator#10664] [Q&A]
Hello, I installed strimzi using helm chart and kafka objects using deployment file. After cluster update pods main-kafka-0 and main-zookeeper-o dissapeer and therre is no chance to recreate it again on that cluster. After deleting ns and recreating it, PVC for zookeper and kafka doesn't want to be created (waiting for first consume). Do you have some idea how this can be solved. Thank you in advance. 

> apiVersion: kafka.strimzi.io/v1beta2
> kind: Kafka
> metadata:
>   namespace: kafka
>   name: main
> spec:
>   kafka:
>     replicas: 1
>     listeners:
>       - name: plain
>         port: 9092
>         type: internal
>         tls: false
>       - name: tls
>         port: 9093
>         type: internal
>         tls: true
>         authentication:
>           type: tls
>       - name: external
>         port: 9094
>         type: loadbalancer
>         tls: true
>         authentication:
>           type: tls
>         configuration:
>           bootstrap:
>             annotations:
>               cloud.google.com/load-balancer-type: Internal
>           brokers:
>           - broker: 0
>             annotations:
>               cloud.google.com/load-balancer-type: Internal
>     storage:
>       type: jbod
>       volumes:
>       - id: 0
>         type: persistent-claim
>         size: 200G
>         deleteClaim: false
>     config:
>       offsets.topic.replication.factor: 1
>       transaction.state.log.replication.factor: 1
>       transaction.state.log.min.isr: 1
>   zookeeper:
>     replicas: 1
>     storage:
>       type: persistent-claim
>       size: 10G
>       deleteClaim: false
>   entityOperator:
>     topicOperator: {}

**Comment by scholzj** [comment]
You would have to provide the full logs from the operator, the configuration in a way where it is properly formatted and readable etc. Without that, nobody can tell you what is happening in your cluster.

**Reply by nevenara1234** [reply]
Hi, I attached logs.
[kafka-logs.txt](https://github.com/user-attachments/files/17229710/kafka-logs.txt)
[kafka-pvc.txt](https://github.com/user-attachments/files/17229725/kafka-pvc.txt)
Tnx

**Reply by scholzj** [reply]
The log is not complete. So it is hard to say what the root issue might be from it. You have to share the full logs. If we stick with what you shared ... then it says quite clearly that:

```
Exceeded timeout of 300000ms while waiting for Pods resource main-zookeeper-0 in namespace kafka to be ready
```

So, the Pod is not ready ... what is is doing? What state is it in? Etc.
